components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/exception/CookieValidationFailedException.java

```diff
@@ -0,0 +1,12 @@
+package org.wso2.carbon.identity.application.authentication.framework.exception;
+
+import java.io.IOException;
+
+/**
+ * Session nonce Cookie Validation Failed Exception.
+ */
+public class CookieValidationFailedException extends IOException {
+    public CookieValidationFailedException(String errorCode, String message) {
```

Comment:
new line after the class definition

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/4509/commits/fbb5f9065181ab04c773a345028bfe9ebdba916f